### DISCUSSION TITLE
```SamplesApp.Skia.Wpf``` build error Uno.UI.Tasks source generator

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#14569] [Q&A] [answered]
Hello, Uno team!
I'm try to build SamplesApp.Skia.Wpf, but some problem is appears for me:

```
src\SourceGenerators\Uno.UI.Tasks\Content\Uno.UI.Tasks.targets(92,11): error MSB4064: The "OutputPath" parameter is not supported by the "ResourcesGenerationTask_v0" task loaded from assembly: Uno.UI.Tasks.v0, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null from the path: src\SourceGenerators\Uno.UI.Tasks\bin\Debug_Shadow\Uno.UI.Tasks.v0.dll. Verify that the parameter exists on the task, the <UsingTask> points to the correct assembly, and it is a settable public instance property.

src\SourceGenerators\Uno.UI.Tasks\Content\Uno.UI.Tasks.targets(89,3): error MSB4063: The "ResourcesGenerationTask_v0" task could not be initialized with its input parameters.
```
![image-1](https://github.com/unoplatform/uno/assets/69314237/32aa395b-06f0-439e-945c-34d4e35f3008)
![image](https://github.com/unoplatform/uno/assets/69314237/481c0125-bad7-4749-93f9-600e72065e42)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This may related to https://github.com/unoplatform/uno/issues/14189. A workaround is to force a rebuild the uno.ui.tasks project first.

I haven't been able to reproduce it reliable, and if you're able to get a binlog of this error, it would help a lot. Thanks!
